src/app-settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "dataSourceId": "DemoApplicationDataSource",
+  "applicationId": "4483c9b0-d505-46c9-a157-94c79f4d7a6a"
```

Comment:
These should be combined into one "absolute ref".
The dataSourceId should not be needed for anything else than fetching this first AppSettings entity.